doc/mkdocs/docs/api/basic_json/at.md

```diff
@@ -70,7 +87,8 @@ Strong exception safety: if an exception occurs, the original value stays intact
 
 1. Constant
 2. Logarithmic in the size of the container.
-3. Constant
+3. See 2.
```

Comment:
Nitpicky, but I'd rather repeat `Logarithmic in the size of the container.` here. The level of indirection for that short sentence confuses me.